### PR TITLE
Showing rooms to users w faves

### PR DESCRIPTION
**Pull request by jaredpowers**
Take a look at this one, I just changed the campus show page so users could see all the rooms at each campus.  I still need to figure out how the favorites are going to work, which I will continue to work on, but wanted to get this up there for y'all.  There isn't much to it at all, just displayed a few more things and changed the show path to show the room, and added avergage_capacity_use to the seed file.

### CODE REVIEW

**Comment by jaredpowers** [comment]
Alright changed a bit up so the Campus' show page renders a sorted list of rooms.  My thoughts are we can style the first three and highlight them or something.

**Comment by vpatel90** [comment]
👀 merge conflicts

File: db/seeds.rb [hunk 1]
```diff
@@ -41,7 +41,7 @@ def time_floor(time, seconds = 15.minutes)
   campus = Campus.all.to_a.sample
   10.times do
     user = User.all.to_a.select{|u| u.campus.id == campus.id}.sample
-    room = campus.rooms.new(name:Faker::Name.name,location:Faker::Address.city, capacity: rand(10..30), picture_url: Faker::Avatar.image, events_count:0)
+    room = campus.rooms.new(name:Faker::Name.name,location:Faker::Address.city, capacity: rand(10..30), picture_url: Faker::Avatar.image, events_count:0, average_capacity_use: rand(1..100))
```
Comment (jah2488) [inline db/seeds.rb:43]: Kind of unrelated to this pull request, but worth noting anyway, if you break up long one liners like this into multiple lines, it slightly cuts down on the chance of a merge conflict if multiple people have to edit the same file.

``` rb
room = campus.rooms.new(name: Faker::Name.name,
                          location: Faker::Address.city, 
                          capacity: rand(10..30), 
                          picture_url: Faker::Avatar.image, 
                          events_count:0, 
                          average_capacity_use: rand(1..100))
```

**Comment by VanAnderson** [comment]
I fixed the merge conflicts and tested this out locally... looks good, let's get it approved and merged.
:+1:

**Comment by Naghmehd** [comment]
👍

**Comment by reckocp** [comment]
👍

**Pull request merged**